client/public/static/locales/pt/common.json

```diff
@@ -1,74 +1,74 @@
-  "DELETE_BUILD": "__STRING_NOT_TRANSLATED__",
-  "DELETE_BUILD_SUCCESS": "__STRING_NOT_TRANSLATED__",
-  "DUPLICATE_DOFUS_OR_TROPHY": "__STRING_NOT_TRANSLATED__",
-  "DUPLICATE_ITEM_IN_SET": "__STRING_NOT_TRANSLATED__",
-  "EQUIPPED": "__STRING_NOT_TRANSLATED__",
-  "EQUIP_ITEMS": "__STRING_NOT_TRANSLATED__",
-  "EQUIP_ITEMS_plural": "__STRING_NOT_TRANSLATED__",
-  "EQUIP_SET": "__STRING_NOT_TRANSLATED__",
-  "ERROR": "__STRING_NOT_TRANSLATED__",
-  "ERROR_OCCURRED": "__STRING_NOT_TRANSLATED__",
-  "INVALID_CHARACTERISTIC_POINTS": "__STRING_NOT_TRANSLATED__",
-  "INVALID_SCROLLED_CHARACTERISTICS": "__STRING_NOT_TRANSLATED__",
-  "ITEMS": "__STRING_NOT_TRANSLATED__",
-  "JOIN_US_DISCORD": "__STRING_NOT_TRANSLATED__",
-  "LANGUAGE": "__STRING_NOT_TRANSLATED__",
-  "LAST_MODIFIED": "__STRING_NOT_TRANSLATED__",
-  "LEVEL": "__STRING_NOT_TRANSLATED__",
-  "LEVEL_ABBREVIATION": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_AP_EXO": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_MP_EXO": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_PRYSMARADITES": "__STRING_NOT_TRANSLATED__",
-  "MULTIPLE_RANGE_EXO": "__STRING_NOT_TRANSLATED__",
-  "MY_BUILDS": "__STRING_NOT_TRANSLATED__",
-  "NEW_BUILD": "__STRING_NOT_TRANSLATED__",
-  "NO_BONUS": "__STRING_NOT_TRANSLATED__",
-  "NO_BUILDS_MATCHED": "__STRING_NOT_TRANSLATED__",
-  "NO_ITEMS_EQUIPPED": "__STRING_NOT_TRANSLATED__",
-  "NO_PERMISSION": "__STRING_NOT_TRANSLATED__",
-  "NUM_ERROR": "__STRING_NOT_TRANSLATED__",
-  "NUM_ERROR_plural": "__STRING_NOT_TRANSLATED__",
-  "NUM_ITEMS": "__STRING_NOT_TRANSLATED__",
-  "OK": "__STRING_NOT_TRANSLATED__",
-  "OR": "__STRING_NOT_TRANSLATED__",
-  "OWNER": "__STRING_NOT_TRANSLATED__",
-  "REPLACE": "__STRING_NOT_TRANSLATED__",
-  "RESET_ALL": "__STRING_NOT_TRANSLATED__",
-  "RESET_ALL_FILTERS": "__STRING_NOT_TRANSLATED__",
-  "RESTART_BUILD": "__STRING_NOT_TRANSLATED__",
-  "SCROLLED": "__STRING_NOT_TRANSLATED__",
-  "SEARCH": "__STRING_NOT_TRANSLATED__",
-  "SELECT_CLASS": "__STRING_NOT_TRANSLATED__",
-  "SELECT_CLASS_DETAILED": "__STRING_NOT_TRANSLATED__",
-  "SELECT_ITEM_TO_REPLACE": "__STRING_NOT_TRANSLATED__",
-  "SETS": "__STRING_NOT_TRANSLATED__",
-  "STATS_PLACEHOLDER": "__STRING_NOT_TRANSLATED__",
-  "SUCCESS": "__STRING_NOT_TRANSLATED__",
-  "UNTITLED": "__STRING_NOT_TRANSLATED__",
-  "VIEW_ITEMS": "__STRING_NOT_TRANSLATED__",
-  "VIEW_SETS": "__STRING_NOT_TRANSLATED__",
-  "WEAPON_AND_SPELLS": "__STRING_NOT_TRANSLATED__"
+  "ACTIONS": "Ações",
+  "ANONYMOUS": "Anónimo",
+  "AVAILABLE_AT_LEVEL": "Disponível no nível {{level}}",
+  "BACK": "Voltar",
+  "BASE": "Base",
+  "BUY_US_BUBBLE_TEA": "Compre-nos um bubble tea",
+  "BUY_US_COFFEE": "Compre-nos um café",
+  "CANCEL": "Cancelar",
+  "CHARACTERISTICS": "Características",
+  "CLEAR_STATS": "Redefina as características base e scroladas para 0",
+  "CONDITION_NOT_MET": "As condições para este item não foram cumpridas.",
+  "CONDITION_NOT_MET_WITH_ITEM": "As condições para o item <1>{{itemName}}</1> não foram cumpridas.",
```

Comment:
https://github.com/samlee405/dofus-lab/commit/e144c79658725b8e1f58db4d9ddd2c95a1953870

This is a bug I fixed yesterday, could you change it to `<2>{{itemName}}</2>` here?